portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLAppHelperLocalServiceImpl.java

```diff
@@ -883,6 +879,31 @@ protected <T extends RepositoryModel<T>> void triggerRepositoryEvent(
 		}
 	}
 
+	private boolean _addDraftAssetEntry(DLFileVersion dlFileVersion) {
```

Comment:
Shouldn't this be names `_isAddDraftAssetEntry`? The method is not adding anything.